main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"github.com/Shopify/sarama"
+	"github.com/companieshouse/chs.go/kafka/consumer/cluster"
+	"github.com/companieshouse/chs.go/kafka/klog"
+	"github.com/companieshouse/chs.go/log"
+	"github.com/companieshouse/company-search-consumer/config"
+	gologger "log"
+	"os"
+)
+
+func main() {
+	cfg := config.Get()
+
+	log.Namespace = "company-search-consumer"
+	if len(os.Getenv("HUMAN_LOG")) < 1 {
+		klg := &klog.Klog{OutTopic: cfg.LogTopic, BrokerAddr: cfg.StreamingBrokerAddr, SchemaRegistryURL: cfg.SchemaRegistryURL}
+		go klg.Capture()
+	}
+	log.Trace("company-search-consumer starting...", log.Data{"config": cfg})
+
+	// Push the Sarama logs into our custom writer
+	sarama.Logger = gologger.New(&log.Writer{}, "[Sarama] ", gologger.LstdFlags)
+
+	var resetOffset bool
+	if cfg.InitialOffset != -1 {
+		resetOffset = true
+	}
+
+	consumerConfig := &consumer.Config{
+		Topics:       []string{cfg.StreamCompanyProfileTopic},
+		ZookeeperURL: cfg.StreamingZookeeperURL,
+		BrokerAddr:   cfg.StreamingBrokerAddr,
+	}
+	groupConfig := &consumer.GroupConfig{
+		GroupName:   cfg.ConsumerGroupName,
+		ResetOffset: resetOffset,
+		Chroot:      cfg.ZookeeperChroot,
+	}
+
+	log.Debug("creating consumer group")
+
+	consumer := consumer.NewConsumerGroup(consumerConfig)
+	if err := consumer.JoinGroup(groupConfig); err != nil {
+		log.ErrorC("Error joining consumer group", err, nil)
+		return
+	}
```

Comment:
I would advise adding a log message to notify that the consumer has been successfully initialised